pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/invertedindex/TextIndexHandler.java

```diff
@@ -138,6 +138,7 @@ public void updateIndices(SegmentDirectory.Writer segmentWriter, IndexCreatorPro
     }
     for (String column : columnsToAddIdx) {
       ColumnMetadata columnMetadata = _segmentMetadata.getColumnMetadataFor(column);
+      LOGGER.info("Creating text index from segment: {}, column: {}", segmentName, column);
       if (shouldCreateTextIndex(columnMetadata)) {
         createTextIndexForColumn(segmentWriter, columnMetadata, indexCreatorProvider, indexCreatorProvider);
```

Comment:
Can we add an INFO log whenever processExistingSegments() returns false? That will make it easy for anyone to debug the whole segment journey.